src/Pages/MembershipPlan/App.css

```diff
@@ -18,3 +18,56 @@
   height: 200px;
   width: 100%;
 }
+.emailVerified{
```

Comment:
for this file, please keep whitespace between css blocks, and remove any whitespace within css blocks